nlp_primitives/number_of_hashtags.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import IntegerNullable, NaturalLanguage
+
+from .count_string import CountString
+
+
+class NumberOfHashtags(CountString):
+    """Determines the number of hashtags in a string.
+
+    Description:
+        Given list of strings, determine the number of hashtags
+        in each string.
+
+        A hashtag is defined as a string that meets the following criteria:
+            - Starts at the start of a string or after whitespace
+            - Contains a sequence of alphanumeric characters
+                - The hashtag must contain at least one alphabetic character
```

Comment:
Combine these: `Contains a sequence of alphanumeric characters with at least one alphabetic character`